File: C++/1145.cpp
#include <iostream>

using namespace std;

int main(){
    
    int limiteLinha, limiteColuna;
    
    cin >> limiteLinha;
    cin >> limiteColuna;
    
    int n = 1;
    
    for(int i = 1; i <= limiteColuna; i++){
            if(n == limiteLinha) {
                 cout << i << "\n";
                 n = 1;
            }
            else{
            cout << i << " ";
            n++;
            }
    }
    return 0;
}

File: JavaScript/1145.js
var input = require('fs').readFileSync('stdin', 'utf8');

var lines = input.split('\n');

let entrada = lines[0].split(' ')
let limiteLinha = parseInt(entrada.shift())
let limiteColuna = parseInt(entrada.shift())

let contador = 0
let str = ""
for (i = 1; i < limiteColuna; i++){
    contador += 1
    if (contador == limiteLinha) {
        contador = 0
        str += `${i}\n`
    }else{
        str += `${i} `
    }

}
str += `${limiteColuna}`
console.log(str)
